### PR TITLE
Pr 3

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
SonarQube has detected issues

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```
Comment (ghost) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAIMWB7p5suXGVi2ZUD###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_3@9
The return value of "length" must be used.

Comment (ghost) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAIMWB7p5suXGVi2ZUE###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_3@9
A "NullPointerException" could be thrown; "message" is nullable here.

**Review by sns-seb2 — CHANGES_REQUESTED** [review]
SonarQube has detected issues

File: src/main/java/org/sonarqube/A.java (hunk 1 above)
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAIMWB7p5suXGVi2ZUE###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_3@9
A "NullPointerException" could be thrown; "message" is nullable here.

Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWAIMWB7p5suXGVi2ZUD###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_3@9
The return value of "length" must be used.

**Review by sns-seb2 — CHANGES_REQUESTED** [review]
Foo

**Review by sns-seb2 — APPROVED** [review]
bar

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:8]: sdsds

**Review by sns-seb2 — DISMISSED** [review]

**Pull request closed without merging**